optuna/visualization/_utils.py

```diff
@@ -26,6 +32,29 @@ def is_available() -> bool:
     return _plotly_imports._imports.is_successful()
 
 
+def _check_plot_args(
+    study: Union[Study, Sequence[Study]],
+    target: Optional[Callable[[FrozenTrial], float]],
+    target_name: str,
+) -> None:
+    if isinstance(study, Study):
+        studies = [study]
+    else:
+        studies = list(study)
```

Comment:
[nits]
I thought `studies = study` was sufficient as `study` is either `Study` or `Sequence[Study]`.
(If you want to explicitly indicate list conversion, just ignore it)